doc/mgr/rgw.rst

```diff
@@ -139,3 +139,32 @@ Join an existing realm by creating a new secondary zone (using the realm token)
   ceph rgw admin [*]
 
 RGW admin command
+
+Automatic usage log trimming
+----------------------------
+
+Enable automatic usage log trimming for one or more RGW realms by
+setting the config option ``usage_trim_realms`` to a comma separated
+list of realms or to an asterisk (``*``) for all realms.
+
+Set the ``usage_trim_older_than_days`` config option to the amount of
```

Comment:
IMHO I find the parameter `usage_trim_older_than_days` missing some information. What about something like `realm_log_retention_days`? I think it's more explicit in this case.

BTW: I think we have to remove the prefix `usage` from parameters and code. We normally don't use such prefix when naming conf params.

Reply:
I understand, the problem is though that since this module is generic to rgw and not per-se a module specific to usage log trimming it causes us to mix a lot of context.

This functionality is explicitly to run the ``radosgw-admin usage trim`` command which clean ``usage`` data objects in the RGW log pool. So when we say "realm log" there really is no such thing, we could say "RGW log trim" but that as well is misleading because we are trimming usage log and not for example replication log.

Do you see my point? I might have explained it a little bit messy but hopefully my point came across.

Reply:
@tobias-urdin thanks for adding more context related to reason behind including `usage` in the names. With that in mind I think it's OK to keep the prefix :+1: